lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -961,7 +976,7 @@ class _PointerAdapter extends _BaseAdapter with _WheelEventListenerMixin {
     });
 
     // Why `domWindow` you ask? See this fiddle: https://jsfiddle.net/ditman/7towxaqp
-    _addPointerEventListener(domWindow, 'pointermove', (DomPointerEvent event) {
+    _addPointerEventListener(_globalTarget, 'pointermove', (DomPointerEvent event) {
```

Comment:
(Rant) I wonder if the solution to all this "listening to global events craze" is to have:

* A per-view listener that is targeted to the view itself for normal mouse moves
* A global listener on window that we can tap from as needed in special cases (when the mouse leaves the window, etc...)

That way we might simplify the general case of pointer handling :/

Reply:
We do need to figure this out at some point, yes. For example, do we care about hover events happening outside flutter views? What if the pointer is down and moves outside of the view? etc.